pages/linux/gcov.md

```diff
@@ -0,0 +1,29 @@
+# gcov
+
+> Code coverage analysis and profiling tool. Discover untested parts of your program.
+> Get copy of source code annotated with execution frequencies of code segments.
+> More information: <https://linux.die.net/man/1/gcov>.
+
+- Generate coverage report named `file.cpp.gcov`:
+
+`gcov {{path/to/file.cpp}}`
+
+- Write individual execution counts for every basic block:
+
+`gcov {{-a|--all-blocks}} {{path/to/file.cpp}}`
```

Comment:
We just use the long argument format:
```suggestion
`gcov {{--all-blocks}} {{path/to/file.cpp}}`
```

Reply:
Not sure we need token syntax on the `--all-blocks` there?